android/src/main/java/com/twiliovoicereactnative/Storage.java

```diff
@@ -38,5 +39,6 @@ static void releaseCallInviteStorage(String uuid, String callSid, int notificati
     Storage.callInviteCallSidUuidMap.remove(callSid);
     Storage.uuidNotificationIdMap.remove(uuid);
     Storage.callAcceptedPromiseMap.remove(uuid);
+    Storage.callRejectPromiseMap.remove(uuid);
```

Comment:
Will there be a problem if the `uuid` does not exist as a key in any of these maps? For example, only either `Storage.callAcceptedPromiseMap` or `Storage.callRejectPromiseMap` will actually have a key-value mapping with that `uuid` since those actions are mutually exclusive, right? These methods won't throw or anything?

Reply:
No, there wont...
`the previous value associated with key, or null if there was no mapping for key. (A null return can also indicate that the map previously associated null with key.)`

https://docs.oracle.com/javase/8/docs/api/java/util/HashMap.html#remove-java.lang.Object-